.claude/commands/amelia/gemini-respond.md

```
@@ -0,0 +1,45 @@
+---
+description: respond to gemini-code-assist review comments on the current PR after evaluation and fixes
+---
+Respond to the gemini-code-assist[bot] review comments on the current PR.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)